plutus-use-cases/test/Spec/Crowdfunding.hs

```diff
@@ -1,264 +1,51 @@
 {-# LANGUAGE DataKinds           #-}
 {-# LANGUAGE NamedFieldPuns      #-}
 {-# LANGUAGE OverloadedStrings   #-}
-{-# LANGUAGE RecordWildCards     #-}
 {-# LANGUAGE ScopedTypeVariables #-}
-{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE TemplateHaskell     #-}
 {-# OPTIONS_GHC -fno-warn-incomplete-uni-patterns -fno-warn-unused-do-bind #-}
 module Spec.Crowdfunding(tests) where
 
-import           Control.Monad                                         (void)
-import           Data.Either                                           (isRight)
-import           Data.Foldable                                         (traverse_)
-import qualified Data.Map                                              as Map
-import           Hedgehog                                              (Property, forAll, property)
-import qualified Hedgehog
+import qualified Spec.Lib                                              as Lib
 import           Test.Tasty
-import           Test.Tasty.Hedgehog                                   (testProperty)
 import qualified Test.Tasty.HUnit                                      as HUnit
 
-import qualified Spec.Lib                                              as Lib
-
-import           Wallet                                                (PubKey (..))
-import           Wallet.Emulator
-import qualified Wallet.Emulator.Generators                            as Gen
-import qualified Wallet.Generators                                     as Gen
-
-import qualified Language.PlutusTx as PlutusTx
-
-import qualified Language.PlutusTx.Coordination.Contracts.CrowdFunding as CF
-import qualified Ledger
-import           Ledger.Ada                                            (Ada)
+import           Language.Plutus.Contract.Test
+import qualified Language.PlutusTx                                     as PlutusTx
+import           Language.PlutusTx.Coordination.Contracts.CrowdFunding
 import qualified Ledger.Ada                                            as Ada
-import qualified Ledger.Value                                          as Value
 
 w1, w2, w3 :: Wallet
 w1 = Wallet 1
 w2 = Wallet 2
 w3 = Wallet 3
 
 tests :: TestTree
-tests = testGroup "crowdfunding" [
-        testProperty "make a contribution" makeContribution,
-        testProperty "make contributions and collect" successfulCampaign,
-        testProperty "cannot collect money too early" cantCollectEarly,
-        testProperty "cannot collect money too late" cantCollectLate,
-        testProperty "cannot collect unless notified" cantCollectUnlessNotified,
-        testProperty "can claim a refund" canRefund,
-        Lib.goldenPir "test/Spec/crowdfunding.pir" $$(PlutusTx.compile [|| CF.mkValidator ||]),
-        let
+tests = testGroup "crowdfunding"
```

Comment:
It feels like we lost a bunch of tests - can we bring this suite up to parity with the old one?

Reply:
Similar comments for the others.